mpconstants/countries.py

```diff
@@ -1,3 +1,4 @@
+# -*- coding: utf8 -*-
```

Comment:
Doesn't look like it does - tests don't pass without this line. I've copied this one from `mpconstants/payments.py`